core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/operators/ConditionalUnitTest.java

```diff
@@ -0,0 +1,65 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ConditionalUnitTest {
```

Comment:
JUnit 5 does not require public classes.

Reply:
done